source/_components/met.markdown

```diff
@@ -18,29 +18,31 @@ redirect_from:
 
 The `met` platform uses the [Met.no](https://met.no/) web service as a source for meteorological data for your location. The weather forecast is delivered by the Norwegian Meteorological Institute and the NRK.
 
+NOTE: Configuration of the Met.no platform via configuration.yaml is deprecated from version 0.95
```

Comment:
Put it in this [kind of note](https://developers.home-assistant.io/docs/en/documentation_create_page.html#html)